src/lib/mina.ts

```diff
@@ -318,8 +324,37 @@ function LocalBlockchain({
             slot: networkState.globalSlotSinceHardFork.toString(),
           });
         }
-      });
 
+        // actions/sequencing events
+        let latestActionsHash =
+          ledger.getAccount(
+            PublicKey.fromBase58(addr),
+            Ledger.fieldOfBase58(tokenId)
+          )?.zkapp?.sequenceState[0] ?? Events.emptySequenceState();
+
+        let actionList = p.body.sequenceEvents;
+        let eventsHash = Events.hash(
+          actionList.map((e) => e.map((f) => Field(f)))
+        );
+
+        latestActionsHash = Events.updateSequenceState(
+          latestActionsHash,
```

Comment:
latestActionHash is supposed to not change if the actions list is empty. here we always modify it, so this needs to be changed